internal/kafka/internal/workers/cluster_mgrs/deprovisioning_clusters_mgr_test.go

```diff
@@ -0,0 +1,238 @@
+package cluster_mgrs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/onsi/gomega"
+	"github.com/pkg/errors"
+
+	apiErrors "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+var autoScalingDataPlaneConfig = &config.DataplaneClusterConfig{
```

Comment:
can we set it as a variable inside the test method if it is only used in one test? If it is used in more than one test and we keep it as a package level variable can we rename this to be more specific? for example adding `test` at the beginning of the name. The reason in the later case is due to all code in the package share that same namespace so from non-test code we would see `autoScalingDataPlane` config variable which I think is too generic as it is a package level namespace where non test code coexists

Reply:
This is only used for testing. It makes sense to make them test specific. Doing that also avoid the need to modify one test ending up messing other tests because the variable is global